### PR TITLE
Add kuryr-e2e-openstack job to openshift/installer

### PR DESCRIPTION
**Pull request by dulek**
This commit adds a job that will run OpenShift on OpenStack with Kuryr
SDN enabled, i.e. deployment's OpenStack Neutron will be used to provide
networking to pods and OpenStack Octavia to create load balancers for
services.

### CODE REVIEW

File: ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml [hunk 1]
```diff
@@ -1341,6 +1341,68 @@ presubmits:
...
+          value: openstack
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: openshift-installer-master.yaml
+              name: ci-operator-master-configs
+        - name: JOB_NAME_SAFE
+          value: kuryr-e2e-openstack
+        - name: KURYR_ENABLED
+          value: "false"
```
Comment (dulek) [inline ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml]: I am keeping that as "false" on purpose for now - we don't want the rehearsal job to mess with CI cluster yet as we still need some stuff to make it work.

**Comment by petr-muller** [comment]
Stale PR, please adapt and reopen if still needed.

/close

**Pull request closed without merging**